projects/media-viewer/src/lib/bookmark/components/bookmark.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Bookmark, BookmarksPerPage} from '../../viewers/pdf-viewer/side-bar/bookmarks/bookmarks.interfaces';
+import {Component, Input, OnDestroy, OnInit} from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import * as fromStore from '../../store/reducers/reducers';
+import * as fromSelectors from '../../store/selectors/bookmarks.selectors';
+import {Observable, Subscription} from 'rxjs';
+
+@Component({
+  selector: 'mv-bookmark',
+  templateUrl: './bookmark.component.html'
+})
+export class BookmarkComponent implements OnInit, OnDestroy {
```

Comment:
could we rename this component to something like bookmark-icon or similar, to differentiate it from the bookmarks component that represents the actual bookmarks